lib/finch/response.ex

```diff
@@ -7,14 +7,14 @@ defmodule Finch.Response do
 
   defstruct [
     :status,
-    :body,
+    body: "",
     headers: []
   ]
 
   @typedoc """
   A body associated with a response.
   """
-  @type body() :: binary() | nil
+  @type body() :: binary()
```

Comment:
consider just inlining this type: `%Response{..., body: binary(), ...}`. perhaps it's useful to export it, idk, so please feel free to just ignore me. :)